handlers/handlers.go

```diff
@@ -218,6 +241,8 @@ func AuthenticateOrRedirectToLogin(
 }
 
 // Authenticated Handlers
+
+// HandleHomeRequest responds to GET requests with the home page.
```

Comment:
seems a bit unecessary

Reply:
I'm open to all non-empty string suggestions.